css/style.css

```diff
@@ -23,6 +23,26 @@
   padding: 3rem 0;
 }
 
+.title-1 {
+  font-size: calc(1rem + 6vw);
```

Comment:
Use pure `rem` for this. The combination of `rem` with `vw` is strange. Also, `calc()` is a CSS 4 thing - it's only supported by modern browsers.